views.go

```diff
@@ -524,6 +561,11 @@ func (lk *logKeeper) viewTestByBuildIdTestId(w http.ResponseWriter, r *http.Requ
 			lk.logErrorf(r, "Error rendering template: %v", err)
 		}
 	}
+
+	grip.ErrorWhen(
```

Comment:
For the "accessing" stats should we log earlier, in case this errors out before the log can actually be accessed or there is a redirect? Or do we only care about logs that are successfully returned?

Reply:
I went back and forth on this. In the end I concluded that by putting them at the end we're spared double counting requests that are retried. Especially for the s3 cost-estimate it made sense to me to exclude retries since they often result from db connection weirdness and the like.

Reply:
What about the redirects? Are we going to miss those?

Reply:
I think on the contrary, by waiting until the end we'll only count it once. If we were to log on redirects we would count it now and also when the browser loads lobster and lobster gets the log through logkeeper's API.